app/assets/javascripts/ace.js.erb

```diff
@@ -0,0 +1,11 @@
+//= require ace/ace
+
+<% modes = ENV['ACE_EDITOR_MODES'].presence || 'javascript,coffee,markdown,sql,json,yaml,text' %>
```

Comment:
If it's not going to work with Docker, I'm hesitant to add it since it'll probably confuse people. What do you think @dsander?